dvc/cache.py

```diff
@@ -137,3 +137,10 @@ def update(self, cache, suffix=""):
 
         for repo_pair, files in cache.external.items():
             self.external[repo_pair].update(files)
+
+    def __eq__(self, other):
```

Comment:
Why do we need this?

Reply:
For tests.

Reply:
Not a big fan. Test code should be localized there, like `namedcache_eq()` helper.